Sources/Commands/Options.swift

```diff
@@ -238,11 +238,9 @@ public struct SwiftToolOptions: ParsableArguments {
           inversion: .prefixedEnableDisable,
           help: "Enable code coverage")
     var shouldEnableCodeCoverage: Bool = false
-
-    // TODO: Does disable-automatic-resolution alias force-resolved-versions?
     
     /// Use Package.resolved file for resolving dependencies.
-    @Flag(name: [.long, .customLong("disable-automatic-resolution")], help: "Disable automatic resolution if Package.resolved file is out-of-date")
+    @Flag(name: [.long, .customLong("disable-automatic-resolution"), .customLong("only-use-versions-from-resolved-file")], help: "Only use versions from the Package.resolved file and fail resolution if it is out-of-date")
```

Comment:
should we also add a deprecation warning when using the old flag to encourage folks to transition over? or at least adda comment here to say which one is "preferred"

Reply:
Not sure we can know which of the flags was used, maybe @natecook1000 has an idea how to do that without splitting it out into multiple flags.